HuskStandalone/HuskStandalone.py

```diff
@@ -70,7 +70,11 @@ def InitializeProcess(self):
 
     def RenderExecutable(self):
         """Return render executable path"""
-        return self.GetRenderExecutable("USD_RenderExecutable")
+        version = self.GetPluginInfoEntryWithDefault( "Version", "" ).replace( ".", "_" )
+        if version:
+            version = "_"+version
+        exe = self.GetRenderExecutable( "USD_RenderExecutable" + version)
+        return exe

```

Comment:
Can we keep to python PEP styling?
Also, can return result directly and only need to `str.replace` if a version is passed.

```suggestion
        version = self.GetPluginInfoEntryWithDefault("Version", "")
        if version:
            version = "_" + version.replace(".", "_")
        return self.GetRenderExecutable("USD_RenderExecutable" + version)
```